Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift

```diff
@@ -13,7 +13,7 @@ struct BasicAppSettingsView: View {
 
     var body: some View {
         List {
-            NavigationLink(destination: FiatCurrenciesView()) {
+            NavigationLink(destination: FiatCurrenciesView().hidesSystemNavigationBar(false)) {
```

Comment:
unfortunately, we now have to add this modifier in order to handle the navigation bar showing/hiding when going in the 2nd screen from the root screen.